qlog/src/streamer.rs

```diff
@@ -178,6 +179,16 @@ impl QlogStreamer {
         self.add_event_data_with_instant(event_data, now)
     }
 
+    /// Writes an [Event] based on the provided [EventData] to a JSON-SEQ record
+    /// at time [std::time::Instant::now()].
+    pub fn add_event_data_ex_now(
```

Comment:
Maybe a builder pattern would be useful here?

```
streamer.add_event(EventBuilder::now().with_data(event_data).with_ext(ex_data));
or maybe
streamer.add_event(Event::at(instant).data(event_data).ext(ex_data));
```

Reply:
Nice idea but we'll save that for another day.